OpenMeteoTests/WeatherForecastTests.cs

```diff
@@ -119,39 +119,11 @@ public async Task WeatherForecast_With_All_Options_Test()
                 Minutely15 = Minutely15Options.All
             };
 
-            var res = await client.QueryAsync(options);
```

Comment:
Let me know what you think. The tests were failing (locally and on the build server) because there were too many API calls made to the open-meteo API.

I thought that these two tests were really only testing that "All" options were being set correctly, so I removed the sync one and mad ethe other test only test locally (no big API calls).

Reply:
The only other solution I could think of is that we have a chat with open-meteo and see if they can give us an API key for use for the tests - but the tests would fail locally if ran all at once.